frontend/src/components/projectSection/multiProjectCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import ArtworkImage from '../gallery/artworkCard/artworkImage';
+import "./projectCard.css";
+
+interface MultiProjectCardProps {
+    username: string,
+    title: string,
+    description: string,
+    images: string[],
+    projectlink: string,
+}
+
+export default class MultiProjectCard extends Component<MultiProjectCardProps>{
```

Comment:
Just a thought, why don't we combine the ProjectCard into here. And for anything with `this.props.images < 2` we render using the normal ProjectCard method, and everything else we can use the carousel.

Reply:
that's a good idea. I'll try to do that tomorrow :p